aiosnmp/connection.py

```diff
@@ -27,11 +29,15 @@ def __init__(
         *,
         host: str,
         port: int = 161,
+        local_addr: str = None,
```

Comment:
let's use `local_addr: Optional[Tuple[str, int]] = None` and remove `local_port` at all.